### ISSUE TITLE
Upgrade from 1.71.1 to 1.94.1 fails

### ISSUE THREAD

**Issue by rjanossy** [tobybatch/kimai2#376]
Hi,
Upgrade from 1.71.1 to 1.94.1 fails with the following error.

![kimai_error](https://user-images.githubusercontent.com/97611067/165597676-c69f30b9-b2a4-4ce8-b941-a01d39a05b94.jpg)

What can be wrong?
Best Regards,
Robert

**Comment by tobybatch** [comment]
Can I get the details of which image, OS, docker version etc

**Comment by rjanossy** [comment]
Hi,
I installed the latest Ubuntu 22.04 LTS on my server. No docker here.
Than followed the instructions here:
https://www.kimai.org/documentation/fresh-ubuntu-20.html

The only difference was here:
git clone -b **1.71.1** --depth 1 https://github.com/kevinpapst/kimai2.git
I wanted to use the 1.71.1 because we have an older system with this version.

1,71.1 was up and running. Then database was restored from an other machine (Ubuntu 20.04 , Kimai2 1.71.1)
After restoring the DB it was working well.

Later I wanted to upgrade to the latest version, the 1.94.1.
I followed these instructions step-by-step: 
https://www.kimai.org/documentation/updates.html

And ended up with  the reported error.

Regards,
Robert
